charts/base-services/values.yaml

```diff
@@ -12,3 +12,7 @@ issuer:
   acme:
     server: https://acme-v02.api.letsencrypt.org/directory
     email: devops@web3.foundation
+
```

Comment:
can you add here a couple of links on how to generate this secret ?
Guess you used https://github.com/kubernetes/ingress-nginx/blob/main/docs/user-guide/nginx-configuration/annotations.md#authentication and https://github.com/kubernetes/ingress-nginx/blob/main/docs/examples/auth/basic/README.md

Reply:
Comment added, password available in our shared vault